Sources/VaporJWT/LoginMiddleware.swift

```diff
@@ -0,0 +1,42 @@
+import Vapor
+import JWT
+import HTTP
+import Node
+
+/// Parses JWT and creates an ephemeral session,
+/// logging the user in with credentials from the token.
+public final class LoginMiddleware: Middleware {
+    let signer: Signer
```

Comment:
I want to discourage fetching the signer from any source other than `drop.jwtSigner()`